dozer/cogs/management.py

```diff
@@ -0,0 +1,200 @@
+"""General, basic commands that are common for Discord bots"""
+
+import asyncio
+import logging
+import math
+
+from datetime import timezone, datetime
+
+import humanize
+import discord
+from dateutil import parser
+from discord.ext.commands import has_permissions, CommandInvokeError
+
+from ..db import asyncpg
+from ._utils import *
+from .general import blurple
+from .. import db
+
+timezones = {"CDT": "UTC-5", "EST": "UTC-5", "EDT": "UTC-4", "GMT": "UTC-0"}
```

Comment:
I feel like there's a better way to do this.

Reply:
Moved timezone info to a json file that fits the very picky format of parser